modules/yarpc/yarpc.go

```diff
@@ -230,17 +230,17 @@ func (c *dispatcherController) mergeConfigs(name string) (conf yarpc.Config, err
 
 // Creates a new YARPC module and adds a common middleware to the global config collection.
 // The first created module defines the service name.
-func newYARPCModule(
+func newModule(
 	mi service.ModuleCreateInfo,
 	reg registerServiceFunc,
 	options ...modules.Option,
-) (*YARPCModule, error) {
+) (*Module, error) {
 	name := "yarpc"
```

Comment:
So on my "no default names" thing - note that for *newModule`, it's defaulting to `yarpc`, while in `ThriftModule` in thrift.go it's defaulting to `rpc` @madhuravi